src/code42cli/cmds/securitydata.py

```diff
@@ -163,9 +163,20 @@ def clear_checkpoint(state, checkpoint_name):
 @security_data.command()
 @file_event_options
 @search_options
+@click.option(
+    "--or-query", is_flag=True, cls=searchopt.AdvancedQueryAndSavedSearchIncompatible
```

Comment:
I do think there is potential usefulness to not having this restriction, but it probably warrants further discussion and should just stay this way for now. We can discuss further once we pursue "merging" cli options with saved search/advanced search.